sdk/include/reactos/svc.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * PROJECT:     ReactOS Service Host
+ * LICENSE:     BSD - See COPYING.ARM in the top level directory
+ * FILE:        sdk/include/reactos/svc.h
+ * PURPOSE:     Global Header for Service Host
+ * PROGRAMMERS: ReactOS Portable Systems Group
+ */
+/* See https://www.geoffchappell.com/studies/windows/win32/services/svchost/process/globaldata.htm?tx=78 */
+
+#ifndef __SVC_H
+#define __SVC_H
+
+#include <rpc.h>
+#include <ndk/rtlfuncs.h>
+#include <ndk/kdtypes.h>
+
+#define DPFLTR_SVCHOST_ID 28
```

Comment:
So that one should not necessary, because it is already declared in psdk/dpfilter.h. (But then our svchost.h should include that latter header...)

Reply:
Fixed. Thanks @HBelusca. I missed it.
This will be indeed better, because the public header doesn't use that declaration anywhere, only private header does it, so it makes sense to include dpfilter.h only there.
Compilation still succeeds for me.